tests/modules/moduleconfigs.c

```diff
@@ -0,0 +1,136 @@
+#include "redismodule.h"
+#include <strings.h>
+int mutable_bool_val;
+int immutable_bool_val;
+long long longval;
+long long memval;
+RedisModuleString *strval;
```

Comment:
we must nullify this otherwise we'll attempt to free an uninitialized pointer, or can't distinguished between old one that should be freed and NULL.
```suggestion
RedisModuleString *strval = NULL;
```